joomla/joomla-cms
# [4.2] Media manager improvements

Pull Request for Issue # .

### Summary of Changes
- add another script in the list of package.json scripts for compiling MM for development (dev tools friendly)
- add the package `dotenv` that is required for the above
- replace the abandoned [vuex-persistedstate](https://www.npmjs.com/package/vuex-persistedstate) with [vuex-persist](https://www.npmjs.com/package/vuex-persist)
- minor CS on the entry of the app


### Testing Instructions

- Run the commands `npm ci`, `npm run build:com_media` and `npm run build:com_media:dev` and verify that media manager works on all of them
- Create an article and select an intro image (preferably one that is inside a deep nested folder)
- Observe that the localSession has a value for the `joomla.mediamanager`
- Try to select an image for the full image input and verify that the media manager opened in the previous selected folder (ie the one you chose for the intro image)


### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Link to documentations
Please select:


The new command is documented in the build/README.md
